src/rgw/rgw_admin.cc

```diff
@@ -244,6 +244,8 @@ void usage()
   cout << "  datalog list               list data log\n";
   cout << "  datalog trim               trim data log\n";
   cout << "  datalog status             read data log status\n";
+  cout << "  datalog type               change datalog type\n";
```

Comment:
maybe `change datalog type to --log-type={fifo,omap}`

Reply:
Done.